WooCommerce/Classes/ViewRelated/Dashboard/Settings/About/AboutViewController.swift

```diff
@@ -93,8 +93,6 @@ private extension AboutViewController {
         /// Hence the container view with a defined frame.
         let footerContainer = UIView(frame: CGRect(x: 0, y: 0, width: Int(tableView.frame.width), height: Constants.footerHeight))
         let footerView = TableFooterView.instantiateFromNib() as TableFooterView
-        footerView.footnoteText = footerTitleText
```

Comment:
hmm is this an intentional change to show the same footer on both the Settings screen and Settings > WooCommerce (`AboutViewController`)? Right now the WooCommerce About screen has a footer that shows the app version and company info:

`develop` / app store | this PR
--- | ---
![Simulator Screen Shot - iPhone 5s - 2019-07-19 at 15 22 51](https://user-images.githubusercontent.com/1945542/61560057-32951680-aa39-11e9-8316-be051e49c90a.png) | ![Simulator Screen Shot - iPhone SE - 2019-07-19 at 15 23 34](https://user-images.githubusercontent.com/1945542/61560058-32951680-aa39-11e9-826b-62cb0febafe8.png)

If this is the case, we might want to also handle the URL tap event from `UITextViewDelegate` as in `SettingsViewController`